pkg/apis/certmanager/v1/types.go

```diff
@@ -184,3 +184,13 @@ const (
 	UsageMicrosoftSGC       KeyUsage = "microsoft sgc"
 	UsageNetscapeSGC        KeyUsage = "netscape sgc"
 )
+
```

Comment:
This had been removed in v1alpha3 or v1beta1 (can't remember which) but is still used by the controllers.
Perhaps it belongs somewhere internal?